gateway/crates/config/src/apq.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(Debug, Clone, PartialEq, Default, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct AutomaticPersistedQueries {
+    /// If automatic persisted queries should be enabled.
+    pub enabled: bool,
```

Comment:
Sorry I didn't re-read the rest of the PR, but is APQ being off by default intended? Intuitively I would have expected the opposite